lib/gutdata/core/logging.rb

```diff
@@ -1,43 +1,50 @@
 # encoding: UTF-8
 #
-# Copyright (c) 2010-2015 GoodData Corporation. All rights reserved.
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
+require 'rest-client'
```

Comment:
This was the particular error we saw (it stems from the to-be-removed elasticsearch "tire" gem ... but I am worried that re-introducing a non-ruby2.5 compat. version of this gem back into the project will create the same problem)

```
      key not found: :ciphers
/home/app/bundle/ruby/2.5.0/gems/rest-client-1.8.0/lib/restclient/request.rb:163:in `fetch'
/home/app/bundle/ruby/2.5.0/gems/rest-client-1.8.0/lib/restclient/request.rb:163:in `initialize'
/home/app/bundle/ruby/2.5.0/bundler/gems/retire-d251c60af171/lib/tire/http/client.rb:11:in `new'
/home/app/bundle/ruby/2.5.0/bundler/gems/retire-d251c60af171/lib/tire/http/client.rb:11:in `get'
/home/app/bundle/ruby/2.5.0/bundler/gems/retire-d251c60af171/lib/tire/search.rb:142:in `perform'
/home/app/bundle/ruby/2.5.0/bundler/gems/retire-d251c60af171/lib/tire/search.rb:35:in `results'
/home/app/bundle/ruby/2.5.0/bundler/gems/retire-d251c60af171/lib/tire/model/search.rb:105:in `search'
/home/app/bundle/ruby/2.5.0/bundler/gems/retire-d251c60af171/lib/tire/model/search.rb:300:in `search'
./lib/mavenlink/search/workspaces.rb:78:in `search_as_user_control'
```

Reply:
@diclophis just a reminder that this is a public repo so we may want to avoid any Mavenlink specific stuff.

That being said, when we forked this from `gooddata-ruby` and made our modifications, we loosened the constraint for `rest-client`. It used to be locked to `< 1.8.x`, but we migrated it to accept up to version `2.0.x` which is compatible with ruby 2.5

Reply:
Ok, sounds good